### DISCUSSION TITLE
Copy SO data to new server

### DISCUSSION THREAD

**Discussion by BSUNetSec** [Security-Onion-Solutions/securityonion#11736] [Unsupported Versions] [answered]
We are planning on upgrading to SO 2.4 at the end of the year but we are seeing if it is possible to transfer the existing data to another server so we have historical data.

Thanks

**Comment by dougburks** [comment, marked as answer]
The 2.4 migration guide might help:
https://docs.securityonion.net/en/2.4/appendix.html

**Reply by BSUNetSec** [reply]
Thanks doug
